packages/react-devtools-shared/src/backend/fiber/renderer.js

```diff
@@ -599,6 +599,8 @@ export function getInternalReactConstants(version: string): {
       !shouldSkipForgetCheck &&
       // $FlowFixMe[incompatible-type] fiber.updateQueue is mixed
       (fiber.updateQueue?.memoCache != null ||
+        (Array.isArray(fiber.memoizedState) &&
+          fiber.memoizedState[0]?.[REACT_MEMO_CACHE_SENTINEL]) ||
```

Comment:
```suggestion
        (Array.isArray(fiber.memoizedState?.memoizedState) &&
          fiber.memoizedState.memoizedState[0]?.[REACT_MEMO_CACHE_SENTINEL]) ||
```

Reply:
Turns out, this actually has 2 levels of `memoizedState`: one is for Fiber, the other one is for Hook.

I've validated it by manually logging Fiber, and its also used in previous implementation, see line 602